changed formatting and updated plan

bjorn-stevens opened the pull request.

JuliaWindmiller1987 committed:
Adjusts track to meet EC in the north

JuliaWindmiller1987 committed:
Updates forecast

JuliaWindmiller1987 committed:
Includes files for FX and flight support

bjorn-stevens committed:
changed formatting and updated plan

JuliaWindmiller1987 committed:
Includes alternative points for circles and ATR coordination

bjorn-stevens committed:
added method for estimating ec overpass dist/time

bjorn-stevens committed:
minor reformatting to process multiple flights

bjorn-stevens committed:
changed to analyze multiple flights

bjorn-stevens committed:
internal notebook changes

bjorn-stevens committed:
update to flight plan formatting

bjorn-stevens committed:
added on airway between SA and Europe

bjorn-stevens committed:
Adjusts track to meet EC in the north

bjorn-stevens commented:
@lkluft @d70-t I can't make this work.  When I try to resolve the merge conflic, following the command line steps, I can't recreate it on my branch.  Maybe I can go to the original commits and see where the kml file came in... I'll keep trying and might learn something.

lkluft commented:
For me a rebase on to `upstream/main` did the trick (see #24). You can give it a try and ping me if there are issues, or simply merge #24

bjorn-stevens committed:
Added way points

Andrea asked that we add intermediate way points as we pass them
as having more frequent waypoints improves the flight planning.

bjorn-stevens committed:
Update of flight plans with intermediate way points
